### ISSUE TITLE
[Accessibility] Assign a keyboard shortcut key to Focus Terminal Output

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#133876]
CC @isidorn

Currently, neither Focus Terminal Tabs View nor Toggle Terminal puts user focus in the terminal output area; the default landing point is terminal input so screen reader users have to manually press `Shift+Tab` key to find the terminal output area, which is inconvenient.

**Comment by isidorn** [comment]
I think it makes sense to have a command "Focus Terminal Output". Though I would not put a keybinding to it by default. Users could add it if they would want.

Assigning to @Tyriar and @meganrogge as owners of the Terminal. Let me know what you think about this.

**Comment by meganrogge** [comment]
I don't believe we currently have a way to know where the output begins. We have been talking about ways we might be able to make the terminal less of a "black box" in the future in https://github.com/microsoft/vscode/issues/133084

**Comment by jooyoungseo** [comment]
Regardless whether it is currently possible to detect the beginning line of terminal output, there should be a way to assign a new command "Focus Terminal Output" that puts user focus to the `<div class = "xterm-accessibility">` area because that area is currently focusable via tab key. Although I assume that area is active only when accessibility mode is on, please consider adding this feature for screen reader users because screen reader users (including me) need a quick way to check terminal output. Currently, it is inconvenient for us to find this terminal output area.

**Comment by isidorn** [comment]
Thanks @meganrogge 
@jooyoungseo if I understood Megan correctly currently VS Code is not aware of the inner structure of the terminal, thus VS Code does not know what element should be focused.
We could also hear what @Tyriar thinks once he comes back from vacation.
